### PR TITLE
docs: add akka-actor-typed dependency to introduction.md

### PR DESCRIPTION
**Pull request by sgc109**
Adding akka-actor-typed dependency for introduction.md in documentation #3528

### CODE REVIEW

File: docs/src/main/paradox/introduction.md [hunk 1]
```diff
@@ -37,8 +37,9 @@ choose an Akka version to run against and add a manual dependency to `akka-strea
   value1=$akka.version$
   symbol2="AkkaHttpVersion"
   value2="$project.version$"
-  group1="com.typesafe.akka" artifact1="akka-stream_$scala.binary.version$" version1=AkkaVersion
-  group2="com.typesafe.akka" artifact2="akka-http_$scala.binary.version$" version2="AkkaHttpVersion"
+  group1="com.typesafe.akka" artifact1="akka-actor-typed$scala.binary.version$" version1=AkkaVersion
```
Comment (raboof) [inline docs/src/main/paradox/introduction.md:40]:
```suggestion
  group1="com.typesafe.akka" artifact1="akka-actor-typed_$scala.binary.version$" version1=AkkaVersion
```

**Review by raboof — COMMENTED** [review]
Good idea: while it's not always needed, it's what the examples expect, so that would be good to suggest. Needs a '_' before the binary scala version, though.

File: docs/src/main/paradox/introduction.md (hunk 1 above)
Comment (sgc109) [reply docs/src/main/paradox/introduction.md:40]: Oh I missed it :)

**Review by raboof — APPROVED** [review]

**Review by jrudolph — APPROVED** [review]
LGTM, thanks, @sgc109.

**Pull request merged**